src/styles/_class-overrides.scss

```diff
@@ -315,3 +321,137 @@
 .graphic_image--circle {
   border-radius: 50%;
 }
+
+//mobile & tablet view
+@media (max-width: 1023px) {
```

Comment:
I used to have `min-width: units('mobile')`, but then realized that there were multiple things I'd have to reset in desktop view so it seemed to make more sense for now to use `max-width`. However, `max-width: units('mobile')` didn't seem like the best idea since at 1024px some of the desktop specific styles started but mobile styles would also be applied if they weren't over-ridden by the desktop styles. I tried  `max-width: units('mobile') - 1px`, but got an error saying units('mobile') is in rem not px

Reply:
I got the same error too when trying to do something similar. It seems to happen when performing an operation with two or more non-scalar values. You might remember me doing something like `units('1px') * 3` which solved the problem. Try to look into how you can store the values in variables and then perform the operation, or convert the units('mobile') to pixels somehow using an SCSS function we might not know about.

Reply:
It's okay. This is outdated now - I switched back to `min-width: units('mobile')` yesterday in the commit `use min-width for mobile view, update desktop view`